config.go

```diff
@@ -109,10 +109,11 @@ type config struct {
 	AdminIPs           []string `long:"adminips" description:"Expected admin host"`
 	AdminUserIDs       []string `long:"adminuserids" description:"User IDs of users who are allowed to access administrative functions."`
 	MinServers         int      `long:"minservers" description:"Minimum number of wallets connected needed to avoid errors"`
-	EnableStakepoold   bool     `long:"enablestakepoold" description:"Enable communication with stakepoold"`
 	MaxVotedAge        int64    `long:"maxvotedage" description:"Maximum vote age (blocks since vote) to include in voted tickets table"`
 	Description        string   `long:"description" description:"Operators own description of their VSP"`
 	Designation        string   `long:"designation" description:"VSP designation (eg. Alpha, Bravo, etc)"`
+	//deprecated
+	EnableStakepoold bool `long:"enablestakepoold" description:"This option has been deprecated and is not settable"`
```

Comment:
To clarify, in config.go it needs to log a message for the user about the deprecation.  Whenever something gets deprecated, the user needs a strong warning because by definition, the option will be removed in the future and thus break the config.

Reply:
srry I overlooked this comment.  A message is now logged if set true.